internal/runners/config/config.go

```diff
@@ -18,13 +23,103 @@ type primeable interface {
 	primer.Analyticer
 }
 
-func NewConfig(prime primeable) (*Config, error) {
-	return &Config{
+func NewList(prime primeable) (*List, error) {
+	return &List{
 		out: prime.Output(),
 		cfg: prime.Config(),
 	}, nil
 }
 
-func (c *Config) Run(usageFunc func() error) error {
-	return usageFunc()
+type configData struct {
+	Key     string `locale:"key,Key"`
+	Value   string `locale:"value,Value"`
+	Default string `locale:"default,Default"`
+}
+
+type configOutput struct {
+	out     output.Outputer
+	cfg     *config.Instance
+	options []mediator.Option
+	data    []configData
+}
+
+func (c *List) Run(usageFunc func() error) error {
+	registered := mediator.AllRegistered()
+	sort.SliceStable(registered, func(i, j int) bool {
+		return registered[i].Name < registered[j].Name
+	})
+
+	var data []configData
+	for _, opt := range registered {
+		configuredValue := c.cfg.Get(opt.Name)
+		data = append(data, configData{
+			Key:     formatKey(opt.Name),
+			Value:   formatValue(opt, configuredValue),
+			Default: formatDefault(mediator.GetDefault(opt)),
+		})
+	}
+
+	c.out.Print(&configOutput{
+		out:     c.out,
+		cfg:     c.cfg,
+		options: registered,
+		data:    data,
+	})
+
+	return nil
+}
+
+func formatKey(key string) string {
+	return fmt.Sprintf("[CYAN]%s[/RESET]", key)
+}
+
+func formatValue(opt mediator.Option, value interface{}) string {
+	var v string
+	switch opt.Type {
+	case mediator.Enum:
+		return fmt.Sprintf("\"%s\"", value)
+	default:
+		v = fmt.Sprintf("%v", value)
+	}
+
+	if v == "" {
+		return "\"\""
+	}
+
+	if len(v) > 100 {
+		v = v[:100] + "..."
+	}
+
+	if value == opt.Default {
+		return fmt.Sprintf("[GREEN]%s[/RESET]", v)
+	}
+
+	return fmt.Sprintf("[BOLD][RED]%s*[/RESET]", v)
```

Comment:
It looks like I screwed this up a bit in the mock (fixed now), but the color of green/blue is for booleans being true/false. It does not relate to default or not. 

Additionally, we want to highlight values that are defined by the user as bold and with an asterisk. This indicates whether the user has specified a value, it does not specify whether the value also happens to be the same as default.